### PR TITLE
[RFC] Advertise Avocado on autotest.github.io

### PR DESCRIPTION
**Pull request by apahim**
There is a card for Avocado requesting to advertise Avocado
in Autotest site: https://trello.com/c/2LGbcVgc

This is the RFC, hoping for the ACK/NACK and, if ACKed, to
determine the information we should advertise in Autotest site.

Signed-off-by: Amador Pahim apahim@redhat.com

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,6 +63,9 @@
         functions such as qualifying new hardware. It's an open-source project
         under the GPL and is used and developed by a number of organizations,
         including Google, IBM, Red Hat, and many others.</p>
+        <p style="color:red;">Autotest in being discontinued in favor of the next
+        generation testing framework,
+        <a href="http://avocado-framework.github.io/">Avocado Framework</a>.</p>
```
Comment (ldoktor) [inline index.html:68]: Yep, that looks like what I imagined. I'm not native, but I think the English should be improved a bit. (Not to say strictly discontinued, but more-like the maintaners moved to newer project avocado, which has the same purpose, only it's suppose to be cleaner and eventually more powerful).

Comment (apahim) [inline index.html:68]: Agreed. Let's wait for @lmr @clebergnu and @adereis feedback as well.

Comment (adereis) [inline index.html:68]: I like the brevity of your sentence, but "discontinued" may be too strong (I would leave this up to @lmr to decide the wording here, as I guess he's the official maintainer of Autotest). Maybe something like "Please check Avocado, a next generation test automation framework being developed by several of the original Autotest team members."

With that said, I suggest a notice be added to multiple places:
- http://autotest.github.io/
- README.rst (autotest.git)
- documentation/source/index.rst (autotest.git)

Comment (lmr) [inline index.html:68]: @apahim,  I like @adereis's suggested sentence - let's go with that.

**Pull request closed without merging**